resources/prompts/explainPolicy.md

```diff
@@ -0,0 +1,4 @@
+@azure Below is an APIM policy file. Explain what the policies do. If any of the `inbound`, `backend`, `outbound` or `on-error` element is empty or only contains a base policy, skip explaining it. If the policy contains C# code block, briefly summarize what the C# code block does instead of explaining the code line by line.
```

Comment:
shall we check whether Azure Agent extension is installed? Or the `@azure` string has no side effect?

Reply:
`@azure` will take no effect if azure agent is not installed.